packages/salesforcedx-vscode-core/src/util/configUtil.ts

```diff
@@ -70,3 +75,7 @@ export class ConfigUtil {
     return undefined;
   }
 }
+
+export function withoutQuotes(item: AnyJson) {
```

Comment:
Can we make this method name more descriptive? also, is this file the best place for this utility?